README.md

```diff
@@ -17,23 +17,19 @@ server_id =
 expire_logs_days =
 ```
 
-And also on the slave:
-
-```ini
-[mysqld]
-server_id =
-```
-
 ## Role Variables
 
 | Variable | Description | Default |
 |----------|-------------|---------|
 | `mysql_replication_gtid` | Whether or not mysql is using gtid for the replication | `true` |
-| `mysql_replication_role` | Defines whether a host is a `master` or a `slave` | mandatory |
 | `mysql_replication_user` | Defines the replication user | mandatory |
 | `mysql_socket` | MySQL Unix domain socket used for connections | `{{ __mysql_socket }}` |
 | `mysql_replication_master` | IP address or FQDN of the master | mandatory (for all slave servers)|
 | `mysql_master_user_gtid` | The value passed to [`master_use_gtid`][master-use-gtid] | `slave_pos` |
+| `mysql_master_user_gtid` | The value passed to [`master_use_gtid`][master-use-gtid] | `slave_pos` |
+| `mysql_replication_role_primary` | Define if the mysql server is primary | `false` |
+| `mysql_replication_role_secondary` | Define if the mysql server is primary | `false` |
```

Comment:
is secondary

Reply:
I just changed  `mandatory (for all slave servers)` to `mandatory (for all replica servers)`